app/services/to_cocina/work/description_mapper.rb

```diff
@@ -54,6 +57,15 @@ def note_params
           end
         end.presence
       end
+
+      def event_params
+        [].tap do |params|
+          if publication_date.year.present?
+            params << CocinaDescriptionSupport.event_date(type: 'publication', primary: true,
+                                                          date: publication_date.to_edtf_s)
+          end
+        end.presence
```

Comment:
I'm wondering if we should prefer the `tap`-based approach or something different, like: 

```suggestion
        return if publication_date.year.blank?

        Array(
          CocinaDescriptionSupport.event_date(type: 'publication', primary: true,
                                              date: publication_date.to_edtf_s)
        ).presence
```

Reply:
In the future, we will be adding additional events, some of which are multiples (e.g., multiple deposit dates), for which I don't think that `Array()` will work as well.